config/default.yml

```
@@ -333,6 +333,17 @@ Rails/MailerName:
   Include:
     - app/mailers/**/*.rb
 
+Rails/MatchRoute:
+  Description: >-
+                  Don’t use match to define any routes unless there is a need to map multiple request types
+                  among [:get, :post, :patch, :put, :delete] to a single action using the :via option.
```

```suggestion
                  among [:get, :post, :patch, :put, :delete] to a single action using the `:via` option.
```

And I think such this detail explanation would help understanding if written in the class comment documentation :-)